assets/src/design-system/components/search/input/index.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo, forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Input,
+  InputContainer,
+  ClearIcon,
+  ChevronIcon,
+  ChevronDecoration,
+  SearchDecoration,
+  SearchIcon,
+  ClearButton,
+} from './components';
```

Comment:
Possibly breaking storybook from circular imports. Getting that error when playing with the input in `Design System/Search/Input/Default`
![image](https://user-images.githubusercontent.com/22185279/107294011-da15d780-6a29-11eb-972c-6d2743480c20.png)

Reply:
<image src="https://user-images.githubusercontent.com/22185279/107294312-6cb67680-6a2a-11eb-9965-188224fdc066.gif" width="250px" />

Reply:
Thanks! I accidentally messed up the `action` helper which was the reason behind this, will resolve with my next push.